.github/workflows/ci.yaml

```diff
@@ -1,8 +1,10 @@
 name: CI
 
 on:
-  - push
-  - pull_request
+  push:
```

Comment:
This should probably be in a separate PR

Reply:
Agreed.  Moved to #16